api/src/test/java/bisq/api/http/service/auth/ApiPasswordManagerTest.java

```diff
@@ -0,0 +1,391 @@
+package bisq.api.http.service.auth;
+
+import bisq.api.http.exceptions.UnauthorizedException;
+
+import bisq.core.app.BisqEnvironment;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.jetbrains.annotations.NotNull;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+
+import com.github.javafaker.Faker;
+
+public class ApiPasswordManagerTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    private String dataDir;
+    private BisqEnvironment bisqEnvironmentMock;
+    private TokenRegistry tokenRegistryMock;
+    private ApiPasswordManager apiPasswordManager;
+
+    private static String getRandomPasswordDifferentThan(String otherPassword) {
+        String newPassword;
+        do {
+            newPassword = Faker.instance().internet().password();
```

Comment:
What is that supposed to do? internet() is a weird method name...
As said above I prefer to not add that dependency.

Reply:
Ah sorry, just saw its a test class. So then I dont care so much about that dependency anymore, but still would like to know better what the class provides.